src/ast.rs

```diff
@@ -370,4 +396,38 @@ mod tests {
       Some(&"json".to_string())
     );
   }
+
+  #[test]
+  fn test_analyze_jsdoc_imports() {
+    let specifier = ModuleSpecifier::parse("file:///a/test.js").unwrap();
+    let source = Arc::new(
+      r#"
+/** @module */
+
+/**
+ * Some stuff here
+ * 
+ * @type {import("./a.js").A}
+ */
+const a = "a";
+
+/**
+ * Some other stuff here
+ * 
+ * @param {import('./b.js').C}
+ * @returns {import("./d.js")}
+ */
+function b(c) {
+  return;
+}
+"#
+      .to_string(),
+    );
+    let parser = DefaultSourceParser::new();
+    let result = parser.parse_module(&specifier, source, MediaType::TypeScript);
+    assert!(result.is_ok());
+    let parsed_source = result.unwrap();
+    let dependencies = analyze_jsdoc_imports(&parsed_source);
+    println!("{:?}", dependencies);
```

Comment:
Shouldn't there be an assert here instead of print? There's no expected list of deps in this test case